src/connectors/dispatcher/noop.js

```diff
@@ -14,6 +14,13 @@ module.exports = new class extends Connector {
         }];
     }
 
+    postV2PlaybookRunRequests () {
```

Comment:
@rexwhite As a quick note, this noop.js file can probably be deleted at this point!  the only files that are currently in use are the mock.js and the impl.js files.